### PR TITLE
Update luogu.ts

### PR DESCRIPTION
**Pull request by LuohanXiao**
[#19](https://github.com/piterator-org/luogu-discussion-archive/issues/19)

[#20 ](https://github.com/piterator-org/luogu-discussion-archive/issues/20)

### CODE REVIEW

File: packages/viewer/src/lib/luogu.ts [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
   `https://cdn.luogu.com.cn/upload/usericon/${user}.png`;
 
 export const getForumUrl = (forum: Forum) =>
-  `https://www.luogu.com.cn/discuss/lists?forumname=${forum.slug}`;
+  `https://www.luogu.com/discuss/lists?forumname=${forum.slug}`;
 
 export const getForumName = (forum: Forum) => forum.name;
 
-export const judgementUrl = "https://www.luogu.com.cn/judgement";
+export const judgementUrl = "https://www.luogu.com/judgement";
```
Comment (wxh06) [inline packages/viewer/src/lib/luogu.ts:21]: `https://www.luogu.com/judgement` 能正常访问吗？

File: packages/viewer/src/lib/luogu.ts [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
 
 const isHTTPorHTTPS = (url: URL) =>
   (url.protocol === "https:" && ["", "443"].includes(url.port)) ||
   (url.protocol === "http:" && ["", "80"].includes(url.port));
 
 const isLuoguUrl = (url: URL) =>
-  url.hostname === "www.luogu.com.cn" && isHTTPorHTTPS(url);
+  url.hostname === "www.luogu.com" && isHTTPorHTTPS(url);
```
Comment (wxh06) [inline packages/viewer/src/lib/luogu.ts:28]: 这个函数的作用是判断 URL 是否为洛谷内链，这么修改是否不妥？

**Review by wxh06 — CHANGES_REQUESTED** [review]
请确保自己知道自己在修改什么，不要仅仅简单地进行全文替换来水 PR。

**Review by bohanjun — CHANGES_REQUESTED** [review]
The changes you made is just simply replace all legacy luogu domains into new domains in our viewer. It's not hard to guess why you haven't also changed these domains in our archive module. The only possible reason I guess is that you are completely unfamiliar to our project. Thus I suggest you carefully read throughout our codes, and then you may pr again.

**Comment by wxh06** [comment]
Fixed in #22.

**Pull request closed without merging**